### DISCUSSION TITLE
Is there any chance that fan control will be able to change rgb one day ?

### DISCUSSION THREAD

**Discussion by jejeAKAgg** [Rem0o/FanControl.Releases#2379] [General]
Hi, 

I have Lian-Li Fans and like a lot of customers having these fans, they are confronted with software conflict between L-Connect and other control panel software (personnally, I have conflict with MSI Center, which controls my MSI AIO)... To get rid of L-Connect, I've found your software which totally erease the conflict software problem thanks to the "Start at user log in" timer. But the only inconvenient with that is that I can't control the rgb of my fans anymore and so they stay at a default pattern color.... By any chance, will fan control introduce one day a rgb editor ? I guess I could edit my rgb on your software as we can import the lianli rgb dll plugin...

Thanks in advance

Note : I've already tested SignalRGB and OpenRGB : one asks me to close all my apps that I need and the other one doesn't detect my unifan hub (incompatibility)

**Comment by Fr0stX76** [comment]
Even if Rem0o would have plans for RGB, which I would assume he does'nt, as it's a whole different beast, it would take months, even years to implement...

I solved this with OpenRGB, [this](https://www.amazon.ca/gp/product/B08K3TYWP6) ans two of [those](https://www.amazon.ca/gp/product/B0CF2JZ5PF). Lian li RGB connector is just a JST instead of a normal ARGB in a manner of keeping you tied to their ecosystem.

**Reply by jejeAKAgg** [reply]
So I would need to buy an adaptater to "bypass" the hub and connect all my fans directly to the motherboard ?

**Reply by Fr0stX76** [reply]
I just looked into the fans... Of course V2 uses a new connector. So this solution only work for fans v1

**Reply by Fr0stX76** [reply]
The fans to the Razer hub, then the hub to the motherboard on an internal usb 2.0 header. This hub is well supported by OpenRGB

**Reply by Fr0stX76** [reply]
And yes, you could just buy the adaptor and connect to the mb argb, but all rgb would be tied to the same channel, so one effect/color for all. I suggested the razor as it gives more argb channels

But you need to have lian li fan v1 for this to work, as v2 uses a lian li propretary connector

**Reply by Rem0o** [reply]
@Fr0stX76 indeed RGB is a whole different beast, and have no plan to get into that, since https://gitlab.com/CalcProgrammer1/OpenRGB exists already and is doing a great job at it.
